src/lib/utils/meta_table_manager.cpp

```diff
@@ -110,7 +110,7 @@ std::string MetaTableManager::_trim_table_name(const std::string& table_name) {
 
 std::vector<std::vector<AllTypeVariant>> MetaTableManager::_materialize_values(
     const std::shared_ptr<const Table>& values) {
-  auto performance_warning_disabler = PerformanceWarningDisabler{};
+  auto pwd = PerformanceWarningDisabler{};
```

Comment:
?

Reply:
Use of `PerformanceWarningDisabler` here? We materialize the table via `get_rows()`. Since this path is not performance-critical in any way, I added the disabler like in the print operator.

Renaming to abbreviation? Consistency with all other uses of `PerformanceWarningDisabler`. Can change these to `performance_warning_disabler` though.

Reply:
Did not realize that we use `pwd` elsewhere. Since `pwd` is a command in linux, I'd prefer the longer name.